simul/platform/cliutils.go

```diff
@@ -33,16 +36,24 @@ func Scp(username, host, file, dest string) error {
 // Rsync copies files or directories to the remote host. If the DebugVisible
 // is > 1, the rsync-operation is displayed on screen.
 func Rsync(username, host, file, dest string) error {
-	addr := host + ":" + dest
+	h, p, err := net.SplitHostPort(host)
+	if err != nil && strings.Contains(err.Error(), "missing port in address") {
```

Comment:
I'm not sure about this - what happens if you have a `~/.ssh/config` that redirects to another port? Would this still work?
I'd put the `-p %v` as a string, and either give it, if the port is indicated, or leave it out.

Reply:
The docs are already clear that this is not expected to fully work, and is just an aid for dev work. So I'd like to not make it any more complex, trying to anticipate weird .ssh configs.